scripts/wasm-rebuild/docker-scripts/rebuild_tags.sh

```diff
@@ -213,7 +213,9 @@ ln -sf /emsdk_portable/emscripten/bin/* /usr/local/bin
 rm -rf /src
 ln -sf /root/project /src
 
+echo "Install dependencies and upgrade system packages."
 apt-get -qq update >/dev/null 2>&1
+apt-get -qq upgrade >/dev/null 2>&1
```

Comment:
Do we really need the upgrade? This might make it unstable.

Reply:
Yes. This is an old image and has outdated SSL certs. `wget` could not download boost without it.

Maybe I could update only the certs packages but I'd have to figure out which ones and how - just updating `ca-certificates` was not enough when I tried it back then.